src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -31,11 +31,11 @@ protected function setUp()
     {
         parent::setUp();
 
-        if (!extension_loaded('mongo') && !extension_loaded('mongodb')) {
+        if (!extension_loaded('mongo') && (!extension_loaded('mongodb') || !class_exists('MongoDB\Client', false))) {
```

Comment:
Actually, generally speaking, the `mongo` extension should no longer be used. It is actually flagged as legacy since it does not support MongoDB 3.2 and 3.4, and won't be updated anymore.

Please check the drivers compatibility matrix for more information regarding versions supported (https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#php-driver-compatibility)